device_services/device_services.md

```diff
@@ -0,0 +1,56 @@
+# Device Services
+
+## Generic services
+
+Generic services are each representing an aspect or functionality of a device. They are not specific to a device type or manufacturer.  
+
+- [Basic](/device_services/generic/basic.md)
+- [Output binary switch](/device_services/generic/output_binary_switch.md)
+- [Output level switch](/device_services/generic/output_level_switch.md)
+- [Thermostat](/device_services/generic/thermostat.md)
+- [Water heater](/device_services/generic/water_heater.md)
+- [Meter](/device_services/generic/meter.md)
+- [Virtual meter](/device_services/generic/virtual_meter.md)
+- [Numeric sensor](/device_services/generic/numeric_sensor.md)
+- [Presence sensor](/device_services/generic/presence_sensor.md)
+- [Contact sensor](/device_services/generic/contact_sensor.md)
+- [Alarm](/device_services/generic/alarm.md)
+- [Complex alarm](/device_services/generic/complex_alarm.md)
+- [Inverter](/device_services/generic/inverter.md)
+- [Battery](/device_services/generic/battery.md)
+- [Battery charge control](/device_services/generic/battery_charge_control.md)
+- [Chargepoint](/device_services/generic/chargepoint.md)
+- [Door lock](/device_services/generic/door_lock.md)
+- [User code](/device_services/generic/user_code.md)
+- [Schedule entry](/device_services/generic/schedule_entry.md)
+- [Barrier control](/device_services/generic/barrier_control.md)
+- [Fan control](/device_services/generic/fan_control.md)
+- [Color control](/device_services/generic/color_control.md)
+- [Scene control](/device_services/generic/scene_control.md)
+- [Siren control](/device_services/generic/siren_control.md)
+- [Media player](/device_services/generic/media_player.md)
+- [Sound switch](/device_services/generic/sound_switch.md)
+
+## Product specific services
+
+If product is too specific for its functionalities to be supported by generic services, it may have its own dedicated service. 
+
+- [Doorman](/device_services/specific/doorman.md)
+
+## Meta and utility services
+
+Meta and utility services are not representing a functionality of a device, but rather a feature of the system, adapter or network.
```

Comment:
From Z-Wave perspective OTA, parameters, associations, indicator, time, version are functionalities of device.
So i propose we either change description or just use those as device services(?)

Reply:
It's not part of this PR. I will address it at some point. Though the idea is to group together meta-functionalities and non-meta functionalities.